tools/refguide_check.py

```diff
@@ -981,9 +989,8 @@ def check_doctests_testfile(fname, verbose, ns=None,
     with open(fname, encoding='utf-8') as f:
         text = f.read()
 
-    PSEUDOCODE = set(['some_function', 'some_module', 'import example',
-                      'ctypes.CDLL',     # likely need compiling, skip it
-                      'integrate.nquad(func,'  # ctypes integrate tutotial
+    PSEUDOCODE = set([
+        'some_function', 'some_module', 'import example',
```

Comment:
Why this change?

Reply:
These don't match anything in the code base. (ctypes.cdll would but it's already skipped with # doctest +skip)

Reply:
I think they come from scipy, so let's leave them in

Reply:
^ they do, I'll revert this part of the cl.